internal/bminventory/inventory.go

```diff
@@ -388,7 +388,7 @@ func (b *bareMetalInventory) getUserSshKey(params installer.GenerateClusterISOPa
 	}
 	return fmt.Sprintf(`{
 		"name": "core",
-		"passwordHash": "$6$MWO4bibU8TIWG0XV$Hiuj40lWW7pHiwJmXA8MehuBhdxSswLgvGxEh8ByEzeX2D1dk87JILVUYS4JQOP45bxHRegAB9Fs/SWfszXa5.",
+		"passwordHash": "!",
```

Comment:
To disable password login, remove passwordHash completely.

See https://docs.fedoraproject.org/en-US/fedora-coreos/authentication/

Reply:
Interesting, I imagined that simply removing it might work but I have not seen any similar statement in the more up-to-date docs for [the version we're using](https://coreos.github.io/ignition/configuration-v3_1/) so I didn't even try that.

In any case, I have tested the behaviour of `!` and it seems to work as intended. I also like its explicitness. If you still think it's problematic in anyway please let me know.